PartnerMeeting.Rmd

```diff
@@ -0,0 +1,26 @@
+---
+title: "Partner meeting"
+author: "Vizlab, Data Science Branch, IIDD"
```

Comment:
I like this, I'd add our email addresses to it as well, like "author: "Vizlab, Data Science Branch, IIDD". Contacts: Anthony Martinez (email) and Cee Nell (cnell@usgs.gov)"